src/player/bhv_basic_move.cpp

```diff
@@ -98,14 +98,14 @@ Bhv_BasicMove::execute( PlayerAgent * agent )
     // C2D: Helios 18 Tune removed -> replace with BNN
     // bool helios2018 = false;
     // if (wm.opponentTeamName().find("HELIOS2018") != std::string::npos)
-	// helios2018 = true;
-//    if (std::min(self_min, mate_min) < opp_min){
-//
-//    }else{
-//        if (Bhv_BasicBlock().execute(agent)){
-//            return true;
-//        }
-//    }
+    // helios2018 = true;
+    //    if (std::min(self_min, mate_min) < opp_min){
+    //
+    //    }else{
+    //        if (Bhv_BasicBlock().execute(agent)){
+    //            return true;
+    //        }
+    //    }
```

Comment:
this line should be changed

double pressing_level = static_cast<double>(Setting::i().moving_pressing_level) / 100.0;